metrics/web/lib/common/presentation/metrics_theme/model/light_metrics_theme_data.dart

```diff
@@ -123,6 +125,14 @@ class LightMetricsThemeData extends MetricsThemeData {
             titleTextStyle: TextStyleConfig.popupTitleStyle,
             subtitleTextStyle: TextStyleConfig.popupSubtitleStyle,
           ),
+          tooltipPopupTheme: const TooltipPopupThemeData(
+            backgroundColor: Colors.white,
+            textStyle: TextStyleConfig.popupTitleStyle,
+          ),
+          tooltipIconTheme: const TooltipIconThemeData(
+            color: _textPlaceholderColor,
```

Comment:
It seems like on designs we have `0xFF878799` color here. Also, the text style of the tooltip popup theme has a color different from the color on the design (in light and dark theme data).